src/Panel.tsx

```diff
@@ -41,11 +41,17 @@ export const Panel = ({ active }: PanelProps) => {
     [state, setAddonState]
   );
 
-  const emit = useChannel({
-    [GIT_INFO]: (gitInfo: GitInfo) => {
-      setAddonState({ ...state, gitInfo });
+  const emit = useChannel(
+    {
+      [GIT_INFO]: (gitInfo: GitInfo) => {
+        setAddonState({ ...state, gitInfo });
+        if (gitInfo.uncommittedHash !== state.gitInfo.uncommittedHash) {
+          setIsOutdated(true);
+        }
```

Comment:
I think we need to write the `uncommittedHash` to the build because there are a lots of ways this logic can go wrong:

- What happens when you restart SB
- What happens when you switch branches
- What happens when someone else pushes a commit to the branch and you update

etc